.github/workflows/e2e.yaml

```diff
@@ -162,11 +184,11 @@ jobs:
         with:
           cluster_name: ${{ steps.generate-cluster-name.outputs.CLUSTER_NAME }}
           url: ${{ inputs.workflow_trigger == 'soak' && secrets.SLACK_WEBHOOK_SOAK_URL || secrets.SLACK_WEBHOOK_URL }}
-          suite: ${{ inputs.workflow_trigger == 'soak' && 'soak' || inputs.suite }}
+          suite: ${{ inputs.workflow_trigger == 'soak' && 'soak' || inputs.workflow_trigger == 'private_cluster' && 'private_cluster' || inputs.suite }}
```

Comment:
Can we move this logic into the notify action? That would make it more readable instead of having multi layer turnery